app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java

```diff
@@ -25,6 +26,10 @@ public void onCreate() {
         super.onCreate();
 
         CheckIfAppUpdated();
+
+        // Initialize the fresco plugin.
+        // Should be here instead of each activity
```

Comment:
I'm not sure that this is the place. maybe in launcher activity. use mApp for context